Oracle/pcornet_loader.sql

```diff
@@ -99,6 +99,14 @@ begin
  set pdx='NI'
  where pdx='1';
  
+ update pcornet_cdm.lds_address_history
+ set address_zip5= null
+ where not regexp_like (address_zip5, '\d{5}');
```

Comment:
looks good. https://oracle-base.com/articles/misc/regular-expressions-support-in-oracle